Pipfile

```diff
@@ -18,6 +18,7 @@ yaswfp = ">=0.9.3"
 six = ">=1.15.0"
 browser-cookie3 = "==0.11.4"
 importlib_metadata = "==3.7.2"
+brotli = ">=1.0.9"
```

Comment:
I don't see where you are using this but if the objective is being able to decode brotli encoded HTTP requests it seems we should use `httpx[brotli]` as dependency

see https://www.python-httpx.org/#installation

Reply:
`requests` need `brotli` to decode br encoded responses. Many js files are br encoded, that is why i need it. But now we are using `httpx` so we should use `http[brotli]` as you said.

Reply:
But it appears that `httpx[brotli]` does not work on debian... and `brotli` works

Reply:
Indeed I checked with Ubuntu and openSUSE and `pip install brotli` seems to be enough